src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -324,6 +325,7 @@ internal static LanguageVersion RequiredVersion(this MessageID feature)
             {
                 // C# preview features.
                 case MessageID.IDS_FeatureMixedDeclarationsAndExpressionsInDeconstruction:
+                case MessageID.IDS_FeatureCallerArgumentExpression:
```

Comment:
>case MessageID.IDS_FeatureCallerArgumentExpression: [](start = 16, length = 51)

Consider adding a comment that this is a semantic check.

Reply:
Done